infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -126,28 +73,40 @@ public void delete(@Nonnull ConflictTxStream key, Long globalAddress, @Nonnull R
      * @return global address
      */
     public Long getIfPresent(ConflictTxStream conflictKey) {
-        return conflictCache.getIfPresent(conflictKey);
+        return cacheHash.get(conflictKey);
+    }
+
+    private long firstAddress() {
+        if (cacheEntries.peek() == null)
+            return Address.NOT_FOUND;
+        return cacheEntries.peek().txVersion;
     }
 
     /**
-     * Invalidate all records up to a trim mark.
+     * Invalidate the first record with the minAddress.
+     */
+    private void invalidateFirst() {
+        ConflictTxStreamEntry entry = cacheEntries.poll();
+        cacheHash.remove(entry.conflictTxStream);
+        maxConflictWildcard = Math.max(entry.txVersion, maxConflictWildcard);
+        log.trace("Updating maxConflictWildcard. Old = '{}', new ='{}' conflictParam = '{}'.",
+                maxConflictWildcard, entry.txVersion, entry.conflictTxStream);
+    }
+
+    /**
+     * Invalidate all records up to a trim mark (not included).
      *
      * @param trimMark trim mark
-     */
+     * */
```

Comment:
get rid of extra *

Reply:
Please enable auto-formatting with intellij.